pkgs/development/libraries/gstreamer/good/fix_pkgconfig_includedir.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/pkgconfig/meson.build b/pkgconfig/meson.build
+index b3bf0d4d4..89898e57e 100644
+--- a/pkgconfig/meson.build
++++ b/pkgconfig/meson.build
+@@ -3,7 +3,7 @@ pkgconf = configuration_data()
+ pkgconf.set('prefix', get_option('prefix'))
+ pkgconf.set('exec_prefix', '${prefix}')
+ pkgconf.set('libdir', '${prefix}/@0@'.format(get_option('libdir')))
+-pkgconf.set('includedir', '${prefix}/@0@'.format(get_option('includedir')))
++pkgconf.set('includedir', get_option('includedir'))
+ pkgconf.set('GST_API_VERSION', api_version)
+ pkgconf.set('VERSION', gst_version)
+ 
+@@ -20,5 +20,6 @@ foreach p : pkg_files
+   outfile = p + '-1.0-uninstalled.pc'
+   configure_file(input : infile,
+     output : outfile,
+-    configuration : pkgconf)
++    configuration : pkgconf,
++    install_dir : pkg_install_dir)
```

Comment:
Hmm, this is very weird – what does uninstalled mean? Is it not supposed to be installed? Why create it then? They seem to have removed the installed files without any explanation https://github.com/GStreamer/gst-plugins-good/commit/1b3bdf3e10c12532ba363154176d329bada383a2

Reply:
Hmm, the pc file probably is not needed. It is not there on nixpkgs-master either.

Reply:
No the pc file is required. I added the good and bad plugins as dependencies for `editing-services`, but without this fix it could not detect the good plugins.

Reply:
Are you sure? I do not see any `gst-plugins-good` dependency in `gst-editing-services` code.

Reply:
Hmm, while configuring the package it outputs that it searches for the packages.

Reply:
Right, I was grepping for the wrong thing. They are optionally required for tests. Though rather than installing the uninstalled pc file, we should revert GStreamer/gst-plugins-good@1b3bdf3

Reply:
It is only used here: https://github.com/GStreamer/gst-editing-services/blob/1.14.0/tests/check/meson.build#L44

`uninstalled` is definitely useless since it will just point to a build directory.

Reply:
This is the bug that removed the pc file: https://bugzilla.gnome.org/show_bug.cgi?id=780657 The removal does not make sense to me even with this added context – unless they only want to build tests in `gst-build`, whatever that is.

Reply:
I think I understand it now, this is indeed for `gst-build`, during normal build `GST_PLUGIN_SYSTEM_PATH` (which we set in the setup hook) should be used. You can ignore the dependency not found warnings and not install any .pc file.